checks/evaluation/branch_protection.go

```diff
@@ -104,60 +108,76 @@ func BranchProtection(name string, dl checker.DetailLogger,
 
 func computeNonAdminBasicScore(scores []levelScore) int {
 	score := 0
-	for _, s := range scores {
+	for i := range scores {
+                s := scores[i]
 		score += s.scores.basic
 	}
 	return score
 }
 
 func computeAdminBasicScore(scores []levelScore) int {
 	score := 0
-	for _, s := range scores {
+	for i := range scores {
+                s := scores[i]
 		score += s.scores.adminBasic
 	}
 	return score
 }
 
 func computeNonAdminReviewScore(scores []levelScore) int {
 	score := 0
-	for _, s := range scores {
+	for i := range scores {
+                s := scores[i]
 		score += s.scores.review
 	}
 	return score
 }
 
 func computeAdminReviewScore(scores []levelScore) int {
```

Comment:
these function could be re-written with go generic now that they are supported ;)